source/common/scheduler.c

```diff
@@ -1,9 +1,17 @@
 #include "common.h"
 
+#ifndef _WIN32
+#include <poll.h>
+#endif
+
 typedef struct _WaitableEntry
 {
 	Remote *               remote;
+#ifdef _WIN32
 	HANDLE                 waitable;
+#else
+	int                    waitable;

```

Comment:
`./source/common/compat_types.h:typedef	int		HANDLE;`

Will remove, along with the other fixes.